index.php

```diff
@@ -922,6 +919,15 @@ function updateCheckboxAndCookie(layer) {
                 layer_ais.on("change:visible", (evt) => {
                     updateCheckboxAndCookie(evt.target);
                     document.getElementById("license_marine_traffic").style.display = evt.target.getVisible() ? 'inline' : 'none';
+                    
+                    // Refresh position every 2 minutes. marine traffic requests are cached for 2 minutes.
```

Comment:
The comment does not seem to match the code anymore.

Reply:
fixed